### PR TITLE
Added cafcass user access to C8 document

### PR DESCRIPTION
**Pull request by khurshedk**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `keep-helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/private-law/json/AuthorisationCaseField/C8Document/AuthorisationCaseField.json [hunk 1]
```diff
@@ -27,6 +27,20 @@
     "UserRole": "caseworker-privatelaw-judge",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "10/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "c8Document",
+    "UserRole": "caseworker-privatelaw-externaluser-viewonly",
+    "CRUD": "CRUD"
```
Comment (AlokDatta) [inline definitions/private-law/json/AuthorisationCaseField/C8Document/AuthorisationCaseField.json:35]: Only R please

File: definitions/private-law/json/AuthorisationCaseField/C8Document/AuthorisationCaseField.json [hunk 2]
```diff
@@ -27,6 +27,20 @@
...
+  },
+  {
+    "LiveFrom": "10/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "c8WelshDocument",
+    "UserRole": "caseworker-privatelaw-externaluser-viewonly",
+    "CRUD": "CRUD"
```
Comment (AlokDatta) [inline definitions/private-law/json/AuthorisationCaseField/C8Document/AuthorisationCaseField.json:42]: Only R please

File: definitions/private-law/json/AuthorisationCaseField/C8Document/AuthorisationCaseField.json (hunk 1 above)
Comment (khurshedk) [reply definitions/private-law/json/AuthorisationCaseField/C8Document/AuthorisationCaseField.json]: @AlokDatta  Fixed in latest build

File: definitions/private-law/json/AuthorisationCaseField/C8Document/AuthorisationCaseField.json (hunk 2 above)
Comment (khurshedk) [reply definitions/private-law/json/AuthorisationCaseField/C8Document/AuthorisationCaseField.json]: @AlokDatta  fixed in latest build

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**